stakingModel/test/GoodReserveCDai.test.js

```diff
@@ -445,11 +445,12 @@ it("should set marketmaker in the reserve by avatar", async () => {
     let nom = new BN(2e14).toString();
     let denom = new BN(1e15).toString();
     let actual = await contribution.calculateContribution(marketMaker.address, goodReserve.address, founder, cDAI.address, 1e4);
-    expect(actual.toString()).to.be.equal((1e4 - 1e4 * nom / denom).toString());
+    expect(actual.toString()).to.be.equal((1e4 * nom / denom).toString());
   });
 
   it("should be able to sell gd to cDAI with contribution of 20%", async () => {
     let amount = 1e4;
+    let expectedReturn = 800000; // return minus contribution = 1e4 GD = 1000000 * 80% = 800000
```

Comment:
I still dont understand this. Is the reserve ratio 1 here?
if we are selling 10k GD why do we expected 8K cDAI?

Reply:
yes the rr is 100%
added more details
18399f793dd34475cf277effc433a76f204b0070